src/vsr.zig

```diff
@@ -234,18 +236,16 @@ pub const Header = extern struct {
     pub fn calculate_checksum(self: *const Header) u128 {
         const checksum_size = @sizeOf(@TypeOf(self.checksum));
         assert(checksum_size == 16);
-        var target: [32]u8 = undefined;
```

Comment:
I think we carry forward a potential alignment bug that's been latent here for some time, also into our new `checksum()` function (good abstraction!).

Since we're casting an array of u8 bytes (align 1) to a u128 (align 16), I think we're just lucky so far?

CC'ing @ifreund.

Reply:
I will change it to

```zig
pub fn checksum(source: []const u8) u128 {
    var target: u128 = undefined;
    std.crypto.hash.Blake3.hash(source, std.mem.asBytes(&target), .{});
    return target;
}
```

Reply:
> Since we're casting an array of u8 bytes (align 1) to a u128 (align 16), I think we're just lucky so far?
> 
> CC'ing @ifreund.

There's no problem here since `@bitCast()` has no alignment requires as it uses value semantics.

Reply:
Thanks @ifreund